src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -124,7 +127,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
   const goToChatScreen = (channel: ChannelList, from) => {
     setChat({
       ...chat,
-      selectedChannel: channel
+      selectedChannel: channel as unknown as null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The line `selectedChannel: channel as unknown as null` seems incorrect. You're casting `channel` to `null` which doesn't make sense. If you want to set `selectedChannel` to `null`, you should just assign `null` directly.
```diff
-      selectedChannel: channel as unknown as null
+      selectedChannel: null
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->